### PR TITLE
Removing 20-99 check for persons, as gervimenn have lower values.

### PR DESCRIPTION
**Pull request by saevarma**
## What

Removing 7th and 8th number between 20-99 check in isPerson.

## Why

This checks breaks validation for many Gervimenn registered in National Registry's database.
Hey have 7th and 8th number values lower than 20.

As the usage of Gervimenn is getting used more for testing, specially related to GDPR, this package should support them and validate them correctly as persons.

### CODE REVIEW

**Comment by saevarma** [comment]
@HermannBjorgvin this is causing us some issues in the island.is repo. If you could look into this as soon as possible, it would be much appreciated 🙏

**Comment by HermannBjorgvin** [comment]
@saevarma I've tried before to find a list of Gervimenn and the rules for them. Could you provide some detail on how they work?

The merge looks good, will merge it into the next beta update and try to have it published today.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "kennitala",
-  "version": "2.0.0-beta.1",
+  "version": "2.0.0-beta.3",
```
Comment (HermannBjorgvin) [inline package.json:3]: Please change to "2.0.0-beta.2"

File: dist/kennitala.min.d.ts [hunk 2]
```diff
@@ -6,7 +6,7 @@ declare module "kennitala" {
     export function sanitize(kennitala: string | number): string;
     export function format(kennitala: string | number, spacer?: string): string;
 
-    export function generatePerson(date: Date, startingIncrement?: number): string | undefined;
```
[the remark this answers is not in the document]
Comment (saevarma) [reply dist/kennitala.min.d.ts:9]: I see. I'm wondering what is the use case by offering the optional startingIncrement?

[the remark this answers is not in the document]
Comment (HermannBjorgvin) [reply dist/kennitala.min.d.ts:9]: Took some vacation time, apologies for the late reply.

@eirikurn I like that, makes sense to do it that way. Could we put a function overload in the type declaration file?

**Comment by HermannBjorgvin** [comment]
> > @saevarma I've tried before to find a list of Gervimenn and the rules for them. Could you provide some detail on how they work?
> > The merge looks good, will merge it into the next beta update and try to have it published today.
> 
> Þjóðskrá have published this: https://opingogn.is/dataset/pjooskra-gervigogn

Thanks for that link! I remember searching for it sometime without any luck.

File: dist/kennitala.min.d.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (HermannBjorgvin) [reply dist/kennitala.min.d.ts:9]: Perfect, thanks for that. Will merge and publish a new beta version now.

**Review by HermannBjorgvin — APPROVED** [review]

**Comment by HermannBjorgvin** [comment]
@saevarma here we have it https://www.npmjs.com/package/kennitala/v/2.0.0-beta.2

Thanks for the valuable contribution, very much appreciate it!

**Comment by saevarma** [comment]
> @saevarma here we have it https://www.npmjs.com/package/kennitala/v/2.0.0-beta.2
> 
> Thanks for the valuable contribution, very much appreciate it!

Thank you! Keep up the good work 🙏

**Pull request merged**